apps/cli-daemon/src/app/workspace/workspace.service.ts

```diff
@@ -15,10 +15,14 @@ import {
   Logger,
   NotFoundException,
 } from '@nestjs/common';
+import { JSONSchema7 } from 'json-schema';
+import { Draft07 } from 'json-schema-library';
+import * as angularSchema from 'node_modules/@angular/cli/lib/config/schema.json';
```

Comment:
This file should be read in runtime. Currently you're bundling & reading our installed angular pkg, instead you need to read the installed angular pkg in the currently active user's workspace by using node's fs module (readFile, readFileSync).

After this change you can revert the changes you made in the tsconfig file